org.eclipse.xtext.doc/contents/310_eclipse_support.html

```diff
@@ -656,10 +656,54 @@ <h2 id="project-wizard">Project Wizard</h2>
 
 <p>In the <code>src</code> folder of the <code>ui</code> project a <code>MyDslNewProjectWizardInitialContents</code> Xtend file will also be generated, where you can specify the initial contents that your language wizard will generate.</p>
 
+
```

Comment:
you can find the original source for documentation here: https://github.com/eclipse/xtext/tree/website-master/xtext-website/documentation

Reply:
thanks, I will transfer the changes.